### ISSUE TITLE
`webimage_extra_packages: [webp]` fails when using `nodejs_version: "18"`

### ISSUE THREAD

**Issue by dillonbailey** [drud/ddev#4681]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Run a Diagnostic and Paste Link Here

https://gist.github.com/dillonbailey/f8c24b02cd2d7f124b75e297f83ec0ba

### Expected Behavior

`ddev start` completes successfully

### Actual Behavior

`ddev start` fails with a series of errors related to node18

### Steps To Reproduce

1. With the config from the ddev test above, run `ddev start` 
2. Error output appears in console
3. Downgrading node to version 16 avoids the issue

### Anything else?

Running `sudo apt-get install webp` inside the container works perfectly well once the environment is up and running.

Below is the output from the failed start with node18 and attempting to add webp:

```
failed to solve: executor failed running [/bin/sh -c bash /tmp/setup_node.sh && apt-get install nodejs && npm config set unsafe-perm true && npm install --global gulp-cli yarn]: exit code: 1 
Failed to start autoflip: ComposeCmd failed to run 'COMPOSE_PROJECT_NAME=ddev-autoflip docker-compose -f /Users/dillonbailey/Code/autoflip-craftcms/.ddev/.ddev-docker-compose-full.yaml up --build -d', action='[up --build -d]', err='exit status 17', stdout='#1 [drud/ddev-dbserver-mariadb-10.4:v1.21.4-autoflip-built internal] load build definition from Dockerfile
#1 transferring dockerfile: 591B done
#1 DONE 0.0s

#2 [drud/ddev-webserver:v1.21.4-autoflip-built internal] load build definition from Dockerfile
#2 transferring dockerfile: 1.52kB done
#2 DONE 0.0s

#3 [drud/ddev-webserver:v1.21.4-autoflip-built internal] load .dockerignore
#3 transferring context:
#3 transferring context: 2B done
#3 DONE 0.0s

#4 [drud/ddev-dbserver-mariadb-10.4:v1.21.4-autoflip-built internal] load .dockerignore
#4 transferring context: 2B done
#4 DONE 0.0s

#5 [drud/ddev-dbserver-mariadb-10.4:v1.21.4-autoflip-built internal] load metadata for docker.io/drud/ddev-dbserver-mariadb-10.4:v1.21.4
#5 DONE 0.0s

#6 [drud/ddev-dbserver-mariadb-10.4:v1.21.4-autoflip-built 1/2] FROM docker.io/drud/ddev-dbserver-mariadb-10.4:v1.21.4
#6 DONE 0.0s

#7 [drud/ddev-dbserver-mariadb-10.4:v1.21.4-autoflip-built 2/2] RUN (groupadd --gid 20 "dillonbailey" || groupadd "dillonbailey" || true) && (useradd  -l -m -s "/bin/bash" --gid "dillonbailey" --comment '' --uid 501 "dillonbailey" || useradd  -l -m -s "/bin/bash" --gid "dillonbailey" --comment '' "dillonbailey" || useradd  -l -m -s "/bin/bash" --gid "20" --comment '' "dillonbailey" || useradd -l -m -s "/bin/bash" --comment '' dillonbailey )
#7 CACHED

#8 [drud/ddev-dbserver-mariadb-10.4:v1.21.4-autoflip-built] exporting to image
#8 exporting layers done
#8 writing image sha256:113d7d76fb32566186ee5ef85aee56c32ab2bb11e04e636913a89b9d5e6e85b1 done
#8 naming to docker.io/drud/ddev-dbserver-mariadb-10.4:v1.21.4-autoflip-built done
#8 DONE 0.0s

#9 [drud/ddev-webserver:v1.21.4-autoflip-built internal] load metadata for docker.io/drud/ddev-webserver:v1.21.4
#9 DONE 0.0s

#10 [drud/ddev-webserver:v1.21.4-autoflip-built 1/8] FROM docker.io/drud/ddev-webserver:v1.21.4
#10 DONE 0.0s

#11 [drud/ddev-webserver:v1.21.4-autoflip-built 5/8] RUN mkdir -p /home/dillonbailey && chown dillonbailey /home/dillonbailey && chmod 600 /home/dillonbailey/.pgpass
#11 CACHED

#12 [drud/ddev-webserver:v1.21.4-autoflip-built 6/8] RUN (apt-get remove -y nodejs || true) && (apt purge nodejs || true)
#12 CACHED

#13 [drud/ddev-webserver:v1.21.4-autoflip-built 4/8] RUN export XDEBUG_MODE=off; composer self-update --stable || composer self-update --stable || true; composer self-update --2 || composer self-update --2 || true
#13 CACHED

#14 [drud/ddev-webserver:v1.21.4-autoflip-built 2/8] RUN (groupadd --gid 20 "dillonbailey" || groupadd "dillonbailey" || true) && (useradd  -l -m -s "/bin/bash" --gid "dillonbailey" --comment '' --uid 501 "dillonbailey" || useradd  -l -m -s "/bin/bash" --gid "dillonbailey" --comment '' "dillonbailey" || useradd  -l -m -s "/bin/bash" --gid "20" --comment '' "dillonbailey" || useradd -l -m -s "/bin/bash" --comment '' dillonbailey )
#14 CACHED

#15 [drud/ddev-webserver:v1.21.4-autoflip-built 3/8] RUN apt-get -qq update && DEBIAN_FRONTEND=noninteractive apt-get -qq install -y -o Dpkg::Options::="--force-confold" --no-install-recommends --no-install-suggests webp
#15 CACHED

#16 [drud/ddev-webserver:v1.21.4-autoflip-built 7/8] RUN curl -sSL --fail https://deb.nodesource.com/setup_18.x >/tmp/setup_node.sh ||  curl -sSL --fail https://deb.nodesource.com/setup_18.sh >/tmp/setup_node.sh
#16 CACHED

#17 [drud/ddev-webserver:v1.21.4-autoflip-built 8/8] RUN bash /tmp/setup_node.sh && apt-get install nodejs && npm config set unsafe-perm true && npm install --global gulp-cli yarn
#17 0.201 
#17 0.201 ## Installing the NodeSource Node.js 18.x repo...
#17 0.201 
#17 0.204 
#17 0.204 ## Populating apt-get cache...
#17 0.204 
#17 0.204 + apt-get update
#17 0.372 Hit:1 http://deb.debian.org/debian bullseye InRelease
#17 0.383 Get:2 http://deb.debian.org/debian-security bullseye-security InRelease [48.4 kB]
#17 0.431 Get:3 http://deb.debian.org/debian bullseye-updates InRelease [44.1 kB]
#17 0.616 Get:4 http://deb.debian.org/debian-security bullseye-security/main amd64 Packages [229 kB]
#17 0.724 Get:5 http://deb.debian.org/debian bullseye-updates/main amd64 Packages.diff/Index [16.2 kB]
#17 0.740 Get:6 https://deb.nodesource.com/node_16.x bullseye InRelease [4586 B]
#17 0.753 Get:7 http://deb.debian.org/debian bullseye-updates/main amd64 Packages T-2023-02-20-2009.57-F-2023-02-20-2009.57.pdiff [1070 B]
#17 0.762 Get:7 http://deb.debian.org/debian bullseye-updates/main amd64 Packages T-2023-02-20-2009.57-F-2023-02-20-2009.57.pdiff [1070 B]
#17 1.206 Hit:8 http://nginx.org/packages/debian bullseye InRelease
#17 1.464 Hit:9 https://packages.sury.org/php bullseye InRelease
#17 1.712 Get:10 https://deb.nodesource.com/node_16.x bullseye/main amd64 Packages [777 B]
#17 1.987 Hit:11 http://packages.blackfire.io/debian any InRelease
#17 2.115 Fetched 344 kB in 2s (185 kB/s)
#17 2.115 Reading package lists...
#17 2.769 
#17 2.769 ## Confirming "bullseye" is supported...
#17 2.769 
#17 2.769 + curl -sLf -o /dev/null 'https://deb.nodesource.com/node_18.x/dists/bullseye/Release'
#17 3.239 
#17 3.239 ## Adding the NodeSource signing key to your keyring...
#17 3.239 
#17 3.239 + curl -s https://deb.nodesource.com/gpgkey/nodesource.gpg.key | gpg --dearmor | tee /usr/share/keyrings/nodesource.gpg >/dev/null
#17 3.330 
#17 3.330 ## Creating apt sources list file for the NodeSource Node.js 18.x repo...
#17 3.330 
#17 3.330 + echo 'deb [signed-by=/usr/share/keyrings/nodesource.gpg] https://deb.nodesource.com/node_18.x bullseye main' > /etc/apt/sources.list.d/nodesource.list
#17 3.335 + echo 'deb-src [signed-by=/usr/share/keyrings/nodesource.gpg] https://deb.nodesource.com/node_18.x bullseye main' >> /etc/apt/sources.list.d/nodesource.list
#17 3.340 
#17 3.340 ## Running `apt-get update` for you...
#17 3.340 
#17 3.340 + apt-get update
#17 3.462 Hit:1 http://deb.debian.org/debian bullseye InRelease
#17 3.462 Hit:2 https://packages.sury.org/php bullseye InRelease
#17 3.476 Hit:3 http://deb.debian.org/debian-security bullseye-security InRelease
#17 3.499 Hit:4 http://deb.debian.org/debian bullseye-updates InRelease
#17 3.964 Hit:5 http://nginx.org/packages/debian bullseye InRelease
#17 4.274 Get:6 https://deb.nodesource.com/node_18.x bullseye InRelease [4586 B]
#17 5.092 Hit:7 http://packages.blackfire.io/debian any InRelease
#17 5.230 Get:8 https://deb.nodesource.com/node_18.x bullseye/main amd64 Packages [777 B]
#17 5.252 Fetched 5363 B in 2s (2863 B/s)
#17 5.252 Reading package lists...
#17 5.740 
#17 5.740 ## Run `sudo apt-get install -y nodejs` to install Node.js 18.x and npm
#17 5.740 ## You may also need development tools to build native addons:
#17 5.740      sudo apt-get install gcc g++ make
#17 5.740 ## To install the Yarn package manager, run:
#17 5.740      curl -sL https://dl.yarnpkg.com/debian/pubkey.gpg | gpg --dearmor | sudo tee /usr/share/keyrings/yarnkey.gpg >/dev/null
#17 5.740      echo "deb [signed-by=/usr/share/keyrings/yarnkey.gpg] https://dl.yarnpkg.com/debian stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
#17 5.740      sudo apt-get update && sudo apt-get install yarn
#17 5.740 
#17 5.740 
#17 5.755 Reading package lists...
#17 6.238 Building dependency tree...
#17 6.370 Reading state information...
#17 6.542 The following NEW packages will be installed:
#17 6.542   nodejs
#17 6.644 0 upgraded, 1 newly installed, 0 to remove and 230 not upgraded.
#17 6.644 Need to get 28.6 MB of archives.
#17 6.644 After this operation, 187 MB of additional disk space will be used.
#17 6.644 Get:1 https://deb.nodesource.com/node_18.x bullseye/main amd64 nodejs amd64 18.14.2-deb-1nodesource1 [28.6 MB]
#17 9.219 debconf: delaying package configuration, since apt-utils is not installed
#17 9.273 Fetched 28.6 MB in 2s (12.1 MB/s)
#17 9.309 Selecting previously unselected package nodejs.
(Reading database ... 30566 files and directories currently installed.)
#17 9.370 Preparing to unpack .../nodejs_18.14.2-deb-1nodesource1_amd64.deb ...
#17 9.373 Unpacking nodejs (18.14.2-deb-1nodesource1) ...
#17 13.02 Setting up nodejs (18.14.2-deb-1nodesource1) ...
#17 13.47 npm ERR! `unsafe-perm` is not a valid npm option
#17 13.47 
#17 13.47 npm ERR! A complete log of this run can be found in:
#17 13.47 npm ERR!     /root/.npm/_logs/2023-02-27T01_41_43_857Z-debug-0.log
#17 ERROR: executor failed running [/bin/sh -c bash /tmp/setup_node.sh && apt-get install nodejs && npm config set unsafe-perm true && npm install --global gulp-cli yarn]: exit code: 1
------
 > [drud/ddev-webserver:v1.21.4-autoflip-built 8/8] RUN bash /tmp/setup_node.sh && apt-get install nodejs && npm config set unsafe-perm true && npm install --global gulp-cli yarn:
#17 6.644 After this operation, 187 MB of additional disk space will be used.
#17 6.644 Get:1 https://deb.nodesource.com/node_18.x bullseye/main amd64 nodejs amd64 18.14.2-deb-1nodesource1 [28.6 MB]
(Reading database ... 30566 files and directories currently installed.)
#17 9.370 Preparing to unpack .../nodejs_18.14.2-deb-1nodesource1_amd64.deb ...
#17 9.373 Unpacking nodejs (18.14.2-deb-1nodesource1) ...
#17 13.02 Setting up nodejs (18.14.2-deb-1nodesource1) ...
#17 13.47 npm ERR! `unsafe-perm` is not a valid npm option
#17 13.47 
#17 13.47 npm ERR! A complete log of this run can be found in:
#17 13.47 npm ERR!     /root/.npm/_logs/2023-02-27T01_41_43_857Z-debug-0.log
------
', stderr='failed to solve: executor failed running [/bin/sh -c bash /tmp/setup_node.sh && apt-get install nodejs && npm config set unsafe-perm true && npm install --global gulp-cli yarn]: exit code: 1' 
```

**Comment by rfay** [comment]
This unsafe_perm problem was solved in [v1.21.5-alpha1](https://github.com/drud/ddev/releases/tag/v1.21.5-alpha1), could you please try it out?

**Comment by dillonbailey** [comment]
That did the trick, thanks @rfay !
